### PR TITLE
Update plugin to pull in COSE, JWT, and SD-JWT rendering

### PR DESCRIPTION
**Pull request by decentralgabe**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/241.html" title="Last updated on Feb 26, 2024, 5:54 PM UTC (9b82a7d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/241/921beb0...9b82a7d.html" title="Last updated on Feb 26, 2024, 5:54 PM UTC (9b82a7d)">Diff</a>

### CODE REVIEW

**Review by selfissued — APPROVED** [review]
Thanks for doing this

**Comment by decentralgabe** [comment]
the plugin is working but currently combines COSE, JOSE, and SD-JWT in all tabs. If are OK with that it can be merged soon, which may be a reasonable first step.

if not, I can break out the examples into a few possibilities:
1. All three together (COSE, JOSE, SD-JWT)
2. All three individually
3. ??? whatever we need

**Review by OR13 — APPROVED** [review]

File: plugin/README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Verifiable Credential Service Worker Plugin for ReSpec
 
-But with support for v2, and no JSON-LD processing.
+Builds upon [respec-vc](https://github.com/w3c/respec-vc), but with support for v2, and no JSON-LD processing.
```
Comment (TallTed) [inline plugin/README.md:3]: `[[[VCDM v2]]] [[VCDM v2]]` should get the right citation link, but I don't immediately know the `[[shortcut]]`.
```suggestion
Builds upon [`respec-vc`](https://github.com/w3c/respec-vc), adding support for [[[VCDM v2]]] [[VCDM v2]], and removing JSON-LD processing.
```

Comment (decentralgabe) [reply plugin/README.md:3]: ah this is plain markdown not respect, so will need the real links. I will add them. thank you

**Comment by decentralgabe** [comment]
The change set now incorporates the latest update to the rendering plugin, which allows us to separately render JWT, SD-JWT, and COSE examples.

I've added examples for all three in their respective sections. After #239 is merged this will need to be updated to split out the SD-JWT and JWT examples into their respective sections.

File: plugin/README.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
-# Verifiable Credential Service Worker Plugin for ReSpec
+# Verifiable Credential Service Plugin for ReSpec
 
-But with support for v2, and no JSON-LD processing.
+The
+**[Verifiable Credential Service Worker Plugin for ReSpec](https://github.com/w3c/vc-jose-cose/tree/main/plugin)**
+is basically the **[Verifiable Credential extensions to ReSpec](https://github.com/w3c/respec-vc)**,
+but with support for the [VCDM v2](https://www.w3.org/TR/vc-data-model-2.0), and
+removing JSON-LD processing.
```
Comment (TallTed) [inline plugin/README.md:7]: Either
```suggestion
but adding support for the [VCDM v2](https://www.w3.org/TR/vc-data-model-2.0), and
removing JSON-LD processing.
```

Comment (TallTed) [reply plugin/README.md:7]: or
```suggestion
but with support for the [VCDM v2](https://www.w3.org/TR/vc-data-model-2.0), and
without JSON-LD processing.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by decentralgabe** [comment]
rebased and updated examples per each section. merging.

**Pull request merged**